docs/src/supported_releases.md

```diff
@@ -67,7 +68,7 @@ Git tags for versions are prepended with `v`.
 |-----------------|----------------------|-------------------|---------------------|-------------------------------|---------------------------|-----------------------------|
 | 1.22.x          | Yes                  | December 21, 2023 | ~ July/August, 2024 | 1.26, 1.27, 1.28              | 1.23, 1.24, 1.25          | 12 - 16                     |
 | 1.21.x          | Yes                  | October 12, 2023  | ~ May 23, 2024      | 1.25, 1.26, 1.27, 1.28        | 1.23, 1.24                | 12 - 16                     |
-| main            | No, development only |                   |                     |                               |                           | 11 - 16                     |
+| main            | No, development only |                   |                     |                               |                           | 12 - 16                     |
```

Comment:
We could also populate the "Tested, but not supported" column with all the versions we test and set the other columns to N/A or just `-`.